pkg/monitor/cluster/prometheusalerts.go

```diff
@@ -17,8 +17,16 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/portforward"
 )
 
-var ignoredAlerts = map[string]struct{}{
-	"ImagePruningDisabled": {},
+var ignoredAlerts = map[string]struct{ namespaces map[string]struct{} }{
+	"ImagePruningDisabled": {nil},
+	"KubeJobFailed": {map[string]struct{}{
+		"openshift-azure-logging":  {},
+		"openshift-image-registry": {},
```

Comment:
This is important! This job failing can fill up the disk and cause node failure.

Reply:
ok, but these job related alerts are very noisy. With that level of noise, the alert is practically unusable, making the actual cases where it is an actual problem invisible

Reply:
Maybe the actual solution is to adjust monitor threshold in Geneva, rather than skipping metric emission. WDYT?

Reply:
well, two aspects there:
- the alert KubeJobFailed and KubeJobCompletion is raised by the Prometheus alert manager at the first failure of such jobs.  So adjusting on the monitor side will not help.
- job.conditions are still being monitored by the RP monitor and metrics pushed. So we could maybe place a different monitor on job.conditions and restrict to some more fine tuned alert to track the symptoms raised by @hawkowl

Reply:
@gvanderpotte @hawkowl 

Created [10981845](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/10981845) to track this new approach